### DISCUSSION TITLE
Pages don't use the first paragraph for the og:description like posts do

### DISCUSSION THREAD

**Discussion by funvill** [mmistakes/minimal-mistakes#3026] [Q&A] [answered]
## Environment

- Minimal Mistakes version: 4.23.0 
- Git repository URL: https://github.com/mmistakes/minimal-mistakes
- GitHub Pages hosted (if yes provide URL to site): https://mmistakes.github.io/minimal-mistakes/

## Expected behavior

A "page" should use the first paragraph as the "og:description" when it `page.excerpt` not defined in the page YAML. This is simlare behavior to the way that a "post" does it. 

## Steps to reproduce the behavior

Using the example installation of https://mmistakes.github.io/minimal-mistakes/

1. Goto a "post" and view the source https://mmistakes.github.io/minimal-mistakes/jekyll/post-exclude-search/ ([source](https://github.com/mmistakes/minimal-mistakes/blob/e134d13e457a71900f1d5732cb8668205627ba66/test/_posts/2017-11-28-post-exclude-search.md)) as expected the first paragraph is used as the `og:description`. 

`<meta property="og:description" content="This post should not appear in the search index because it has the following YAML Front Matter:">`

2. Goto a "page" and view the source https://mmistakes.github.io/minimal-mistakes/terms/ ([source](https://github.com/mmistakes/minimal-mistakes/blob/e134d13e457a71900f1d5732cb8668205627ba66/test/_pages/terms.md)) instead of the first paragraph being used the [site's description](https://github.com/mmistakes/minimal-mistakes/blob/e134d13e457a71900f1d5732cb8668205627ba66/docs/_config.yml#L18) is used. 

`<meta property="og:description" content="A flexible Jekyll theme for your blog or site with a minimalist aesthetic.">`

3. The same issue occurs on this "page" as well https://mmistakes.github.io/minimal-mistakes/page-a/ ([source](https://github.com/mmistakes/minimal-mistakes/blob/e134d13e457a71900f1d5732cb8668205627ba66/test/_pages/page-a.md))

`<meta property="og:description" content="A flexible Jekyll theme for your blog or site with a minimalist aesthetic.">`

## notes 

@iBug Hopefully this describes my issue in more detail.

**Comment by mmistakes** [comment]
I'll just leave this here.
https://jekyllrb.com/docs/pages/#excerpts-for-pages

Jekyll doesn't automatically generate an excerpt for pages unless you're on a newer version and enable it. Otherwise you need to set `excerpt: Whatever you want the excerpt to be` in your page or collection doc's front matter.

**Comment by mmistakes** [comment, marked as answer]
Manually setting `excerpt` does work as can be seen in this page from the demo site's docs.

Source MD: https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_docs/01-quick-start-guide.md
https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/

```html
<meta name="description" content="How to quickly install and setup Minimal Mistakes for use with GitHub Pages.">
```

**Reply by BobTheExplorer** [reply]
How would we change the website description for index.html, nor writing an excerpt in the files yaml or putting <meta name="description" content="How to quickly install and setup Minimal Mistakes for use with GitHub Pages."> works

**Comment by funvill** [comment]
Thank you !
